### ISSUE TITLE
Increased CSF homovanillic acid

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7443]
**Preferred term label:**
Increased CSF homovanillic acid

**Synonyms**

**Definition (free text, please give PubMed ID)**
Increased concentration of homovanillic acid (HVA) in the cerebrospinal fluid. HVA is a metabolite of dopamine.
(ALso, abnormal grouping term)

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:613135  PMID:19478460  3/3, infantle onset

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0034201
